README.md

```diff
@@ -153,14 +152,13 @@ security:
             switch_user: false
             logout_on_user_change: true
 
-            form_login:
+            contao_login:
                 login_path: contao_frontend_login
                 check_path: contao_frontend_login
-                default_target_path: contao_index
+                default_target_path: contao_root
+                failure_path: contao_root
```

Comment:
I assume you are sure that this can be a route name instead of a path, too.

Reply:
Yeah, everything can be routes, I was wrong about this the last time. The issue was that we didn't use `httpUtils` to generate the route but directly created our own `RedirectResponse`.